support/node-production-aws-sns/index.js

```diff
@@ -1,11 +1,12 @@
 const { ApiPromise, WsProvider } = require('@polkadot/api');
 const AWS = require('aws-sdk');
 const Settings = require('./settings.json')
+const types = require('../../blockchain/types.json')
```

Comment:
I'm not a fan of relative imports, but configuring for absolute imports seems difficult... Consider researching some ways to make this import independent of where the node-production-aws-sns file is located.

Reply:
Maybe I can request the json file from the repo? But this doesn't seem right... 
`request('https://github.com/trustfractal/protocol/blob/master/blockchain/types.json', (error, response, body) => { ... })`

Reply:
Definitely not. Relative import is much better than a network request and encoding the repository.